README.md

```diff
@@ -25,7 +25,7 @@ $ npm install --save dat.gui
 const dat = require('dat.gui');
 
 // ES6:
-import dat from 'dat.gui';
+import * as dat from 'dat.gui';
```

Comment:
If I'm reading right, shouldn't both `import dat` and `import * as dat` work now?

Reply:
Oh, I see you said this above. Sounds good to recommend the tree-shaking-friendly syntax.